spec/text-editor-component-spec.js

```diff
@@ -411,26 +411,28 @@ describe('TextEditorComponent', () => {
       await component.getNextUpdatePromise()
       const [cursor1, cursor2] = element.querySelectorAll('.cursor')
 
-      expect(getComputedStyle(cursor1).opacity).toBe('1')
-      expect(getComputedStyle(cursor2).opacity).toBe('1')
-
-      await conditionPromise(() =>
-        getComputedStyle(cursor1).opacity === '0' && getComputedStyle(cursor2).opacity === '0'
-      )
-
       await conditionPromise(() =>
         getComputedStyle(cursor1).opacity === '1' && getComputedStyle(cursor2).opacity === '1'
       )
-
       await conditionPromise(() =>
         getComputedStyle(cursor1).opacity === '0' && getComputedStyle(cursor2).opacity === '0'
       )
+      await conditionPromise(() =>
+        getComputedStyle(cursor1).opacity === '1' && getComputedStyle(cursor2).opacity === '1'
+      )
 
       editor.moveRight()
       await component.getNextUpdatePromise()
 
-      expect(getComputedStyle(cursor1).opacity).toBe('1')
-      expect(getComputedStyle(cursor2).opacity).toBe('1')
```

Comment:
Restructuring the earlier assertions seems good, but I'm concerned about dropping these. We want to assert that the cursor is visible for a short duration after moving before resuming blinking. Did you see a specific failure on these assertions or just the ones above? Losing this wouldn't be the end of the world since a regression in cursor blinking behavior wouldn't go unnoticed for long, but it does drop some coverage.